mgmtsystem_information_security/migrations/13.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,12 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    openupgrade.update_module_names(
+        env.cr,
+        [("mgmtsystem_info_security_manual", "mgmtsystem_information_security")],
+        merge_modules=True,
+    )
```

Comment:
This won't work, I think..

On current deployments `mgmtsystem_information_security` won't be installed, so this script will never run.

I'm not sure what the process is to handle this, most probably a script in https://github.com/OCA/openugprade ?

in any case, is it really worth the trouble to rename the module?

Reply:
All right. I will rename ASAP. More simple is more good. Thanks @ivantodorovich

Reply:
Renamed! Thanks